### PR TITLE
added code coverage task

### PR DESCRIPTION
**Pull request by jfelten**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -47,14 +47,25 @@
   "artifacts":
     untracked: true
     
-"unit-tests":
```
Comment (alejandroEsc) [inline .gitlab-ci.yml]: why this name change?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -47,14 +47,25 @@
...
+"build_unit-tests":
   "stage": "build"
   "image": "registry.cnct.io/team-nes/grpc-example/rootfs/golang-protoc"
   "script":
   - "go test server/*test.go"
   - "go test client/*test.go"
 
-"build_image":
+"build_test-coveraage":
```
Comment (alejandroEsc) [inline .gitlab-ci.yml]: coverage

**Comment by alejandroEsc** [comment]
I think 20% code coverage is ok to start with.

File: .gitlab-ci.yml (hunk 1 above)
Comment (jfelten) [reply .gitlab-ci.yml]: trying to be consistent with the company standards by prefixing all build tasks with the appropriate name

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -47,14 +47,25 @@
   "artifacts":
     untracked: true
     
-"unit-tests":
+"build_unit-tests":
   "stage": "build"
   "image": "registry.cnct.io/team-nes/grpc-example/rootfs/golang-protoc"
   "script":
   - "go test server/*test.go"
```
Comment (alejandroEsc) [inline .gitlab-ci.yml]: we can do this via go test ./server/ ./client in one line, may change later on.

**Pull request merged**